src/clients/perf/scenario_perf_client.cpp

```diff
@@ -60,8 +60,8 @@ class ScenarioPerfClient : public Base
         "Sending {} {} transactions", transactions.size(), element_name);
       for (const auto& transaction : transactions)
       {
-        const auto method = transaction["method"];
-        const auto params = transaction["params"];
+        const std::string& method = transaction["method"];
```

Comment:
I'm not sure why this is `const&` assigned? We need to do a conversion, since the righthand side is `nlohmann::json`.

So safer to do:
```suggestion
        const std::string method = transaction["method"];
```

Or:
```suggestion
        const auto method = transaction["method"].get<std::string>();
```

Same for the similar changes elsewhere.